code/modules/mob/mob_helpers.dm

```diff
@@ -22,6 +22,27 @@
 			return 0
 	return 1
 
+/mob/proc/get_screen_colour()
+
+/mob/dead/observer/get_screen_colour()
+	return list()

```

Comment:
Now that the client color always gets updated on login, that should no longer be an issue. If it is, something is wrong elsewhere.